common/txmgr/confirmer.go

```diff
@@ -203,6 +207,17 @@ func (ec *Confirmer[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) sta
 		return errors.Wrap(err, "Confirmer: failed to load EnabledAddressesForChain")
 	}
 
+	ec.tracker.Reset()
+	err = ec.tracker.SetEnabledAddresses(ec.enabledAddresses)
+	if err != nil {
+		return errors.Wrap(err, "Confirmer: failed to set tracker enabled addresses")
+	}
+
+	err = ec.tracker.TrackAbandonedTxes(context.Background())
+	if err != nil {
+		return errors.Wrap(err, "Confirmer: failed to track abandoned transactions")
+	}
```

Comment:
We could maybe use a similar start/close lifecycle like the Confirmer and other sub-components of TXM here. That way you can put this setup logic in a start call and reset in a close call. If that seems like overkill, we can maybe move this to a `Setup` method to simplify keeping the tracker specific logic in the tracker